src/main/java/com/github/manolo8/darkbot/utils/APIErrors.java

```diff
@@ -0,0 +1,70 @@
+package com.github.manolo8.darkbot.utils;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import static com.github.manolo8.darkbot.gui.utils.Popups.showMessageSync;
+
+public class APIErrors {
+
+
+    private final String RUNTIME_URL  = "https://darkbot.eu/downloads/Runtimes4DarkBot.exe";
+    private final String RUNTIME_LINK = "Download link: <a href=\"#\">" + RUNTIME_URL;
```

Comment:
When you open an HTML tag you have to close it afterwards

Also have you tested if by adding an A with an actual href it lets you click it without needing to add a mouse listener?